src/main/java/spoon/support/template/SubstitutionVisitor.java

```diff
@@ -482,4 +488,18 @@ public void scan(CtElement element) {
 		} catch (DoNotFurtherTemplateThisElement ignore) {
 		}
 	}
+
+	public CtElement substitute(CtElement element) {
```

Comment:
So the main entry point of this class is no more `scan` but `substitute` right? It could be good to document that.